java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -63,7 +63,7 @@
     javax.swing.JSpinner spinY;
 
     public static AbstractAction getCoordinateEditAction(final Positionable pos) {
-        return new AbstractAction(Bundle.getMessage("SetXY")) {
+        return new AbstractAction(Bundle.getMessage("SetXY") + "...") {
```

Comment:
The ellipsis should be in the string returned by Bundle.getMessage(). The same is true for all of them (as the ellipsis may not be correct on the right side of a message in all languages).